core/domain/story_domain.py

```diff
@@ -1084,6 +1106,15 @@ def update_thumbnail_filename(self, thumbnail_filename):
         """
         self.thumbnail_filename = thumbnail_filename
 
+    def update_thumbnail_size_in_bytes(self, new_thumbnail_size_in_bytes):
+        """Updates the size of thumbnail of the story.
+
+        Args:
+            new_thumbnail_size_in_bytes: int|None. The new thumbnail size of the
```

Comment:
"The size of the story's new thumbnail."

Having said that -- this actually seems wrong. The thumbnail and its size should always be updated together, concurrently. I don't think you should have separate independent methods for both of those.

Reply:
updated